### PR TITLE
Fix handling of Run Commands from properties.

### PR DESCRIPTION
**Pull request by SteveNewson**
Allow users to source the run commands from properties.

Signed-off-by: Steve Newson <Steven.Newson@gmail.com>

### CODE REVIEW

File: src/main/java/io/fabric8/maven/docker/config/handler/property/PropertyConfigHandler.java [hunk 1]
```diff
@@ -196,12 +196,7 @@ private String extractName(String prefix, Properties properties) throws IllegalA
 
 
     private List<String> extractRunCommands(String prefix, Properties properties) {
-        List<String> ret = new ArrayList<>();
-        List<String> cmds = listWithPrefix(prefix, RUN, properties);
-        if (cmds == null) {
-            return null;
-        }
-        return ret;
```
Comment (rhuss) [inline src/main/java/io/fabric8/maven/docker/config/handler/property/PropertyConfigHandler.java]: Really stupid issue here. Thanks for spotting this !

**Comment by SteveNewson** [comment]
Sure

**Comment by SteveNewson** [comment]
I've reverted the erroneous change.

**Comment by rhuss** [comment]
thanks a lot, lgtm

[merge]

**Pull request merged**